frontend/webapp/containers/main/actions/choose-action-body/custom-fields/pii-masking.tsx

```diff
@@ -12,21 +13,70 @@ type Parsed = {
   piiCategories: string[];
 };
 
+const ListContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  gap: 32px;
+`;
+
+const strictPicklist = [
+  {
+    id: 'CREDIT_CARD',
+    label: 'Credit Card',
+  },
+];
+
+const isSelected = (id: string, selected: string[]) => {
+  return !!selected?.find((str) => str === id);
+};
+
 const PiiMasking: React.FC<Props> = ({ value, setValue }) => {
   const mappedValue = useMemo(() => safeJsonParse<Parsed>(value, { piiCategories: [] }).piiCategories, [value]);
 
-  const handleChange = (arr: string[]) => {
+  const [isLastSelection, setIsLastSelection] = useState(false);
+
+  useEffect(() => {
+    if (!mappedValue.length) {
+      const payload: Parsed = {
+        piiCategories: strictPicklist.map(({ id }) => id),
+      };
+
+      setValue(JSON.stringify(payload));
+      setIsLastSelection(payload.piiCategories.length === 1);
+    }
+    // eslint-disable-next-line
+  }, []);
+
+  const handleChange = (id: string, isAdd: boolean) => {
+    const arr = isAdd ? [...mappedValue, id] : mappedValue.filter((str) => str !== id);
+
     const payload: Parsed = {
       piiCategories: arr,
     };
 
     setValue(JSON.stringify(payload));
+    setIsLastSelection(arr.length === 1);
   };
 
   return (
     <FieldWrapper>
       <FieldTitle>Attributes to mask</FieldTitle>
-      <InputList value={mappedValue} onChange={handleChange} />
+
+      <ListContainer>
+        {strictPicklist.map(({ id, label }) => {
+          const selected = isSelected(id, mappedValue);
+
+          return (
+            <Checkbox
+              key={id}
+              title={label}
+              disabled={isLastSelection && selected}
```

Comment:
you can use Array.includes function and remove the isSelected 
```suggestion
              disabled={isLastSelection && mappedValue.includes(id)}
```